Formula/zsh-fast-syntax-highlighting.rb

```diff
@@ -0,0 +1,29 @@
+class ZshFastSyntaxHighlighting < Formula
+  desc "Feature-rich syntax highlighting for Zsh"
+  homepage "https://github.com/zdharma-continuum/fast-syntax-highlighting"
+  url "https://github.com/zdharma-continuum/fast-syntax-highlighting.git",
+      tag:      "v1.55",
+      revision: "5351bd907ea39d9000b7bd60b5bb5b0b1d5c8046"
+  license "BSD-3-Clause"
+  head "https://github.com/zdharma-continuum/fast-syntax-highlighting.git", branch: "master"
+
+  uses_from_macos "zsh" => [:build, :test]
+
+  def install
+    pkgshare.install Dir["*"]
+  end
+
+  def caveats
+    <<~EOS
+      To activate the syntax highlighting, add the following at the end of your .zshrc:
+        source #{HOMEBREW_PREFIX}/share/zsh-fast-syntax-highlighting/fast-syntax-highlighting.plugin.zsh
```

Comment:
If we install this file using `zsh_function.install` then one would only need to add `$HOMEBREW_PREFIX/share/zsh/site-functions` to `FPATH`. If you use Homebrew Zsh you don't even need that (it's already in `FPATH`).

Reply:
I didn't know we had `zsh_function` 😄 . I will definitely consider using it for other Zsh things. However that isn't the objective here.

This is meant to be `source`d once, in the user's config, upon startup. The user might want to do that manually in zshrc (as shown here), or by a plugin manager such as Antigen, Zinit, etc.

Reply:
AH, right. Misread the `.plugin.zsh`.